### PR TITLE
Add types

### PR DESCRIPTION
**Pull request by Rokt33r**

### CODE REVIEW

**Comment by Rokt33r** [comment]
@wooorm @ChristianMurphy Could you review this? To implement types for `hast-util-to-html`, we should implement types of this package first because its options interface is used by `hast-util-to-html`.

**Review by ChristianMurphy — APPROVED** [review]
LGTM, thanks @Rokt33r!

File: types/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// TypeScript Version: 3.0
+
+declare namespace stringifyEntities {
+  interface StringifyEntitiesOptions {
+    /**
+     * Whether to only escape possibly dangerous characters (`boolean`,default: `false`).
```
Comment (wooorm) [inline types/index.d.ts]:
```suggestion
     * Whether to only escape possibly dangerous characters (`boolean`, default: `false`).
```

**Comment by Rokt33r** [comment]
@wooorm I pushed the fix. Please use squash when merging this pr.

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Nice, thanks! Is your plan to work on parse-entities next? I’d prefer to keep them in sync with each other!

**Comment by Rokt33r** [comment]
My plan was working on `hast-util-to-html` and I did this because `hast-util-to-html` is using its option, `StringifyEntitiesOptions`. But yeah, I can change my priority. I'll add it to my todo list here. https://github.com/rehypejs/rehype/issues/27

**Pull request merged**